design/resources.go

```diff
@@ -347,3 +347,25 @@ var _ = Resource("trackerquery", func() {
 	})
 
 })
+
+var _ = Resource("search", func() {
+	BasePath("/search")
+
+	Action("show", func() {
+		Routing(
+			GET(""),
+		)
+		Description("Search work items using the given query")
+		Params(func() {
+			Param("q", String, "q")
+		})
+		Response(OK, func() {
+			Media(CollectionOf(workItem))
```

Comment:
I would need to read up about how to add composite collections here :)
For now, this seems fine though since we support workitems only.